### PR TITLE
Resolve exercise 2

### PR DESCRIPTION
**Pull request by Stefano-loop**

### CODE REVIEW

File: exercise_2/exercise_2.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+require 'rubygems'
+require 'bundler/setup'
+require 'byebug'
+
+def sort_array(vector)
+    vector.sort!{
+        |firstString, secondString|
```
Comment (fede-moya) [inline exercise_2/exercise_2.rb]: Take a look to [this](https://github.com/rubocop-hq/ruby-style-guide#naming-conventions) naming convention, it's the one we use.

File: exercise_2/exercise_2_spec.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+require 'rubygems'
+require 'bundler/setup'
+require 'rspec'
+require './exercise_2'
+
+describe 'sort_array' do
```
Comment (fede-moya) [inline exercise_2/exercise_2_spec.rb]: Take a look to [this](http://www.betterspecs.org/#describe) guide on specs. According to the guide the description should be
```ruby
describe '#sort_array'
```

File: exercise_2/exercise_2_spec.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    it 'Sorts an array based on the length of the words' do
```
Comment (fede-moya) [inline exercise_2/exercise_2_spec.rb]: Please add some border tests cases, like the expected output when an empty array is given. Also, according to the better specs the it description should be something more like 
```ruby
  it 'sorts the array by the items length ascendingly' do 
```

File: exercise_2/exercise_2_spec.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+        expect(sort_array(["a","abc","ac"])).to eq(["a","ac","abc"])
+        expect(sort_array(["azz","abcg","jkl","asdasdd","a"])).to eq(["a","azz","jkl","abcg","asdasdd"])
```
Comment (fede-moya) [inline exercise_2/exercise_2_spec.rb]: Please use one expect for each it block, it's a [suggestion](http://www.betterspecs.org/#single) of the better specs guide.
